experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -479,54 +478,13 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
 
           instrumentation._headerCapture.server.captureRequestHeaders(span, header => request.headers[header]);
 
-          // Wraps end (inspired by:
-          // https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/master/src/instrumentations/instrumentation-connect.ts#L75)
-          const originalEnd = response.end;
-          response.end = function (
-            this: http.ServerResponse,
-            ..._args: ResponseEndArgs
-          ) {
-            response.end = originalEnd;
-            // Cannot pass args of type ResponseEndArgs,
-            const returned = safeExecuteInTheMiddle(
-              () => response.end.apply(this, arguments as never),
-              error => {
-                if (error) {
-                  utils.setSpanWithError(span, error);
-                  instrumentation._closeHttpSpan(span, SpanKind.SERVER, startTime, metricAttributes);
-                  throw error;
-                }
-              }
-            );
-
-            const attributes = utils.getIncomingRequestAttributesOnResponse(
-              request,
-              response
-            );
-            metricAttributes = Object.assign(metricAttributes, utils.getIncomingRequestMetricAttributesOnResponse(attributes));
-
-            instrumentation._headerCapture.server.captureResponseHeaders(span, header => response.getHeader(header));
-
-            span
-              .setAttributes(attributes)
-              .setStatus({ code: utils.parseResponseStatus(SpanKind.SERVER, response.statusCode) });
-
-            if (instrumentation._getConfig().applyCustomAttributesOnSpan) {
-              safeExecuteInTheMiddle(
-                () =>
-                  instrumentation._getConfig().applyCustomAttributesOnSpan!(
-                    span,
-                    request,
-                    response
-                  ),
-                () => { },
-                true
-              );
-            }
-
-            instrumentation._closeHttpSpan(span, SpanKind.SERVER, startTime, metricAttributes);
-            return returned;
-          };
+          // After 'error', no further events other than 'close' should be emitted.
```

Comment:
This comment is a bit misleading. It refers to `close` event but on the next line the `finsish` event is used.